lisa/parameter_parser/argparser.py

```diff
@@ -38,9 +38,10 @@ def support_variable(parser: ArgumentParser) -> None:
         "-v",
         dest="variables",
         action="append",
-        help="Define one or more variables in the format of `name:value`, which will "
+        help="Variables are defined in runbooks, LISA doesn't pre-define any variable, "
+        "Specify one or more variables in the format of `name:value`, which will "
         "overwrite the value in the YAML file. It can support secret values in the "
-        "format of `s:name:value`.",
+        "format of `s:name:value`. Learn more from documents.",
```

Comment:
Maybe we can say that tests may benefit from custom variables to tune their behavior, too.

Reply:
The help is for test case runner, and they need to understand more on which parameter is needed, and how to set them.

Reply:
Oh, now I see that what I said is indeed impossible now, nothing gets saved for unmatched runbook variables. We gotta think better how to have test-level variables, soon. The rest of the PR seems fine, BTW.

Reply:
Hmm if I understand you right, you are talking about any test param variable passed being saved anywhere in LISAv3. I had that concern as well, we should have a way of saving the test params passed.